### PR TITLE
Add docs for https://github.com/vitessio/vitess/pull/13056

### PR DESCRIPTION
**Pull request by austenLacy**
Adds docs for the `--schema-version-max-age-seconds` vttablet flag added in https://github.com/vitessio/vitess/pull/13056.

### CODE REVIEW

File: content/en/docs/17.0/reference/vreplication/flags.md [hunk 1]
```diff
@@ -148,6 +148,27 @@ All vstreams on a tablet share a common engine. vstreams that are lagging might
...
+
+#### watch_replication_stream
+
+**Type** bool\
+**Default** false\
+**Applicable on** source
+
+By default vttablets reload their schema every `--queryserver-config-schema-reload-time` seconds (default 30 minutes). This can cause a problem while streaming events if DDLs are applied on the source and streaming is started _after_ the DDL was applied but _before_ vttablet refreshed its schema. This is alleviated by enabling the _watcher_.
+
+When enabled, vttablet will start the _watcher_ which streams the MySQL replication stream from the local database, and uses it to proactively update its schema when it encounters a DDL.
```
Comment (mattlord) [inline content/en/docs/17.0/reference/vreplication/flags.md]: This is a repeat. It's also there earlier in the file (line 131). We should remove one of them -- perhaps you meant to move it here to be in alphabetical order?

**Review by mattlord — CHANGES_REQUESTED** [review]
Looks great! We just need to remove the unrelated duplicate section.

File: content/en/docs/17.0/reference/vreplication/flags.md (hunk 1 above)
Comment (austenLacy) [reply content/en/docs/17.0/reference/vreplication/flags.md]: Doh, a copy/paste error most likely

**Comment by mattlord** [comment]
Thanks! Now just need to address the DCO failure and we're good! :) https://github.com/vitessio/website/pull/1475/checks?check_run_id=13617586365

**Review by mattlord — APPROVED** [review]
❤️

**Pull request merged**